test/mock/index.js

```diff
@@ -17,9 +17,12 @@
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
+var path = require('path');
 
-module.exports = {
-    channel: require('./channel.js'),
-    logger: require('./logger.js'),
-    membership: require('./membership.js')
-};
+require('glob').sync(__dirname + '/**/*.js').forEach(function(file) {
```

Comment:
nitpick: This style of loading modules makes it difficult to know what it actually is.

Having an object literal makes reading the code very easy, you just read the code.

Seeing a glob and a bunch of meta programming means you have to `ls __dirname + **/*.js` and then figure out what it actually is.